components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -18,6 +18,21 @@
 #import "MaterialElevation.h"
 
 @class MDCActionSheetAction;
+@class MDCActionSheetController;
+
+/**
+ MDCActionSheetControllerDelegate provides a method that allows a delegate of an
+ MDCActionSheetController to respond to its dismissals.
+ */
+@protocol MDCActionSheetControllerDelegate <NSObject>
+@optional
+
+/**
+ This method allows a delegate conforming to MDCActionSheetControllerDelegate to respond to
+ MDCActionSheetController dismissals.
+ */
+- (void)actionSheetControllerDidDismiss:(nonnull MDCActionSheetController *)actionSheetController;
```

Comment:
Should we add a `NS_SWIFT_NAME` here?

Reply:
Open to it, but I'm not sure what I'd change it to. Looking at [UIScrollViewDelegate's scrollViewDidEndScrollingAnimation](https://developer.apple.com/documentation/uikit/uiscrollviewdelegate/1619379-scrollviewdidendscrollinganimati), it looks like the swift name is the default.